plasma_framework/contracts/src/framework/registries/VaultRegistry.sol

```diff
@@ -1,11 +1,16 @@
 pragma solidity ^0.5.0;
 
-import "../modifiers/Operated.sol";
+import "../utils/Operated.sol";
 
 contract VaultRegistry is Operated {
     mapping(uint256 => address) private _vaults;
     mapping(address => uint256) private _vaultToId;
 
+    modifier onlyFromVault() {
+        require(_vaultToId[msg.sender] != 0, "Not being called by registered vaults");
```

Comment:
well....it's rely on `uint` to make it positive. As a result, if a client really tries to register with `-1`, it would still work but just get a super huge number: `115792089237316195423570985008687907853269984665640564039457584007913129639935`.

However, would change the requirement here to be `_vaultToId[msg.sender] > 0`

Reply:
I was thinking to put that requirement (> 0), in register... function.